frontend/tutorial/step_7.md

```diff
@@ -14,17 +14,19 @@ const baseProps = {
 
 describe('Hello', () => {
 
+  const getLocals = Hello.prototype.getLocals;
```

Comment:
right, I forgot that we often reference `this` in `getLocals` for other things that are not `props`. I'll change it.